### PR TITLE
chore(RHINENG-1262): Upgrade React Router

### PR DESCRIPTION
**Pull request by AsToNlele**
[RHINENG-1262](https://issues.redhat.com/browse/RHINENG-1262)

Upgraded react router to v6, fixed some errors in the console

How to test:
1. Make sure all links and redirects work correctly

### CODE REVIEW

File: src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React, { useState, useEffect } from 'react';
-import { useHistory, useParams } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
```
Comment (johnsonm325) [inline src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js:2]: this pulls out the id, but then the id isn't used. Maybe we can drop it?

**Review by johnsonm325 — APPROVED** [review]
lgtm with the small change.

File: src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js (hunk 1 above)
Comment (AsToNlele) [reply src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js:2]: I believe the `id` is still being used here and in the `useEffect`

image

**Comment by AsToNlele** [comment]
Thanks! Let's get one more review

**Review by gkarat — APPROVED** [review]
LGTM! tested locally and couldn't spot any issues 👍🏼  thanks @AsToNlele

**Pull request merged**